### DISCUSSION TITLE
New Payments method

### DISCUSSION THREAD

**Discussion by Hojjatrt** [saleor/saleor#8693] [Q&A]
hey guys, I want to use saleor in **Iran** but we cannot use **PayPal** or another because these are not able in Iran so can you help me how can I start developing a new **payments** method???

**Comment by karolkielecki** [comment]
@Hojjatrt Hi! You can find more info on developing a payment gateway here https://docs.saleor.io/docs/3.0/developer/extending/payment-gateways

You can also check how other payment methods have been implemented, e.g., Adyen or Stripe.
